mllib/src/main/scala/org/apache/spark/ml/feature/Bucketizer.scala

```diff
@@ -54,7 +54,7 @@ final class Bucketizer private[ml] (override val parent: Estimator[Bucketizer])
       "bucket, which also includes y. The splits should be strictly increasing. " +
       "Values at -inf, inf must be explicitly provided to cover all Double values; " +
       "otherwise, values outside the splits specified will be treated as errors.",
-    Bucketizer.checkSplits)
+    Bucketizer.checkSplits _)
```

Comment:
Actually I don't know. Both Scala and Intellij complain if I don't put the underscore to make it anonymous.

~~~
[error] /Users/meng/src/spark/mllib/src/main/scala/org/apache/spark/ml/feature/Bucketizer.scala:57: missing arguments for method checkSplits in object Bucketizer;
[error] follow this method with `_' if you want to treat it as a partially applied function
[error]     Bucketizer.checkSplits)
~~~